app/assets/javascripts/components/story/StoryItem.js

```diff
@@ -104,12 +104,22 @@ const StoryLabels = ({ labels }) => {
   );
 };
 
-const StoryInfo = ({ storyType ,title, labels, estimate }) => (
+export const StoryTitle = ({title, ownedByInitials, ownedByName}) => (
+  <div className="Story__title">
+    {title} <abbr className="Story__initials" title={`${ownedByName}`}>{ownedByInitials}</abbr>
```

Comment:
can we break the line here to place `abbr` on another line?

Reply:
sure we can